src/gasPriceOracle/adapters/ethereum.ts

```diff
@@ -58,37 +49,6 @@ export async function eip1559Raw(
   };
 }
 
-/**
- * @notice Returns fee data using provider's getFeeData() method.
- * @note Resolves priority gas pricing poorly, because the priority fee is hardcoded to 1.5 Gwei in ethers v5's
- * getFeeData() method
- * @dev TODO: Remove this function soon. See note above about slowly rolling out eip1559Raw.
- * @param provider ethers RPC provider instance.
- * @param chainId Chain ID of the provider instance.
- * @returns Promise of gas price estimate object.
- */
-export async function eip1559Bad(
```

Comment:
Maybe a dumb question, but we don't currently use this _anywhere_? No chain relies on it?

Totally fine with reversing the default, but want to make sure we're not breaking anything.

Reply:
in production no, we do not use it at all